projects/ng-qrcode/src/lib/qr-code.component.ts

```diff
@@ -23,6 +23,7 @@ import { QrCodeErrorCorrectionLevel, RGBAColor } from "./types"
     }
   `,
   styles: [],
+  standalone: false,
```

Comment:
This can be true if you include `imports: [QrCodeDirective]` allowing users to import the component as standalone or in the module.
This will also require change in the `NgModule`